### PR TITLE
Beacon state import and list feature

### PR DESCRIPTION
**Pull request by claravanstaden**
Adds features for:
- importing a ssz state file into the beacon store
- listing all the states in the beacon store
- fixes a bug in the beacon store module related to file paths

Needs to be cherry-picked into https://github.com/Snowfork/snowbridge/tree/release-v1.0.0 after merge.

### CODE REVIEW

File: relayer/relays/beacon/store/datastore.go [hunk 1]
```diff
@@ -169,8 +174,51 @@ func (s *Store) WriteEntry(attestedSlot, finalizedSlot uint64, attestedStateData
...
 
+func (s *Store) ListBeaconStates() ([]BeaconState, error) {
+	var response []BeaconState
+
+	query := `SELECT id, attested_slot, finalized_slot, attested_sync_period, finalized_sync_period, attested_state_filename, finalized_state_filename FROM beacon_state ORDER BY attested_slot`
+
+	rows, err := s.db.Query(query)
+	if err != nil {
+		return response, fmt.Errorf("no match found")
+	}
```
Comment (yrong) [inline relayer/relays/beacon/store/datastore.go:185]: Do we want to implement a pagination query here?

**Comment by yrong** [comment]
May not be directly with change in this PR

https://github.com/Snowfork/snowbridge/blob/74124615060a032de59bddf2f91a78686fdba4b3/relayer/relays/beacon/store/datastore.go#L310-L320

Do we want to add an unique constraint on `finalized_slot` to make sure we're not importing duplicate entry into the db?

File: relayer/relays/beacon/store/datastore.go (hunk 1 above)
Comment (claravanstaden) [reply relayer/relays/beacon/store/datastore.go:185]: It is limited to 100 states so I thought it is not necessary yet, but will be cool for the future!

**Comment by claravanstaden** [comment]
> May not be relevant with change in this PR
> 
> https://github.com/Snowfork/snowbridge/blob/74124615060a032de59bddf2f91a78686fdba4b3/relayer/relays/beacon/store/datastore.go#L310-L320
> 
> Do we want to add an unique constraint on `finalized_slot` to make sure we're not importing duplicate entry into the db?

Good idea! :) https://github.com/Snowfork/snowbridge/pull/1189/commits/65d9e6806d1469805bd9ff1d61eba26e6dd01b3f

**Review by vgeddes — APPROVED** [review]

**Pull request merged**